app/views/user_categories/index.html.erb

```diff
@@ -7,14 +7,18 @@
   <body>
   <div class="container">
   <div class="row">
+
     <% @categories.each do |category| %>
       <aside class="col-md-6">
         <h2><%= category.title %></h2>
         <p><%= category.description %></p>
         <p>You've leaned - </p>
```

Comment:
@Kohei-Oki 

- Please update `leaned` to `learned`
- Display the lesson result if not nil
- If result is displayed, change button to redirect  to `lessons#show`